speech/unit_tests/test_client.py

```diff
@@ -199,11 +199,10 @@ def test_sync_recognize_with_empty_results_gax(self):
         credentials = _Credentials()
         client = self._makeOne(credentials=credentials, use_gax=True)
         client.connection = _Connection()
+        _MockGAPICSpeechAPI._results = []
```

Comment:
I can override `GAPICSpeechAPI._gapic_api` instead I think.
But it is this way now because `SpeechApi` gets called in the `GAPICSpeechAPI` constructor. `self._gapic_api = SpeechApi()`.